rascal-vscode-extension/src/RascalLanguageServer.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2018-2021, NWO-I CWI and Swat.engineering
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+import { LanguageClient } from 'vscode-languageclient/node';
+import { activateLanguageClient } from './RascalLSPConnection';
+import { LanguageParameter, TermLanguageServer } from './TermLanguageServer';
```

Comment:
we should pick a term for "TermLanguageServer" or "Language Parameterized Server" and use it everywhere. Now we have two names for the same thing and that becomes confusing. TermLanguageServer is nice and short, but you'd have to come from the ASF+SDF times to understand why it is called like that. Or maybe a third name would be better, like "MultiplexedLanguageServer" or something. The README uses "Language-Parametrized"

Reply:
I agree, the term "term" has run it's course. `ParameterizedLanguageServer` would match better with the java side of the extension. But I also like `MultiplexedLanguageServer`, since that is more a description of what it does.

Reply:
and we'd rename the Java classes too, right? in one go.

Reply:
Optimizing for less invasive file movements, we could use `ParameterizedLanguageServer`? 

Also, I think Generic/Generalized is to "generic" (no pun intended).

Reply:
I'm ok with that. It says what it does.